pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -232,6 +232,7 @@ stdenv.mkDerivation (finalAttrs: {
     ./0015-tpm2_context_init-fix-driver-name-checking.patch
     ./0016-systemctl-edit-suggest-systemdctl-edit-runtime-on-sy.patch
     ./0017-meson.build-do-not-create-systemdstatedir.patch
+    ./0019-Revert-boot-Make-initrd_prepare-semantically-equival.patch
```

Comment:
Should this only apply for aarch64? Or is it possible for x86 and other architectures EFI support could be affected?

Reply:
An AMD Opteron that could theoretically be affected by this was mentioned in the upstream issue, though AFAICT nobody has reported whether it actually is in practice or not. Also conditional patches are just evil and we should avoid platform divergence wherever possible.